syft/frameworks/crypten/hook/hook.py

```diff
@@ -7,6 +7,19 @@
 
 
 class CrypTenPlanBuild(object):
+    """
+    TODO: When the problem converting Onnx serialized models to PyTorch is solved
```

Comment:
Should this TODO info be at this place in the code?

Reply:
I think it is best to be in the ```OnnxModel.py``` file. Moving it.